### PR TITLE
Style the status check page and get from dynamics

### PR DESCRIPTION
**Pull request by lewispb**

### CODE REVIEW

**Comment by lewispb** [comment]
👀

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -7,8 +7,12 @@
 // There is a fair bit to understand about how the 3 projects are related and loaded.
 // Pulling in govuk-elements.scss here will pull in all sass at:
 // https://github.com/alphagov/govuk_elements/blob/master/public/sass/_govuk-elements.scss
+
 @import 'govuk-elements';
 @import 'partials/forms';
 @import 'partials/footer';
 
-@import 'datashift_journey/partials/state_jumper_toolbar';
+@import 'status_check';
```
Comment (autotelik) [inline app/assets/stylesheets/application.css.scss]: might be bundle update required then

File: app/views/status_checks/result.html.erb [hunk 2]
```diff
@@ -1 +1,9 @@
-<%= @result.reference_number %>
+<h2 class='heading-medium'>Application Status for <%= @result.reference_number %></h2>
+
+<div class="progress-container">
+  <ol class="progress-meter">
+    <% @result.stage_map.each_with_index do |(k, v), i| %>
+      <li class="progress-point <%= @result.current_stage_index <= i ? 'todo' : 'done' %>"><%= v %></li>
```
Comment (autotelik) [inline app/views/status_checks/result.html.erb]: perhaps better as helper method on @result ?

**Comment by autotelik** [comment]
no blockers :ship:

File: app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (lewispb) [inline app/assets/stylesheets/application.css.scss]: ah yeah, done thanks

File: app/views/status_checks/result.html.erb (hunk 2 above)
Comment (lewispb) [inline app/views/status_checks/result.html.erb]: yeah you're right, moved out to the model with some added tests.

**Pull request merged**